lib/LaTeXML/Core/Definition/Constructor/Compiler.pm

```diff
@@ -150,6 +150,7 @@ sub translate_constructor {
 sub slashify {
   my ($string) = @_;
   $string =~ s/\\/\\\\/g;
+  $string =~ s/'/\\'/g;     # slashify ', since we're wrapping in ''
```

Comment:
actually, may be this is a good time to remind of [quotemeta](https://perldoc.perl.org/functions/quotemeta.html). I ended up being bitten twice in a separate piece of code ( #1260 ) until I remembered to just defer to Perl's own thorough escapism, and avoid unsuspecting weirdness. It could end up a relief to replace all `slashify` calls (and lookalikes) with that.